mkfs/mkfs.c

```diff
@@ -302,10 +302,14 @@ static void usage(const char *program_name)
 {
     const char *p = strrchr(program_name, '/');
     p = p != NULL ? p + 1 : program_name;
-    printf("Usage: %s [-b boot-image] [-r target-root] image-file < manifest-file\n"
+    printf("Usage: %s [-b boot-image] [-r target-root] [-s image-size] "
+           "image-file < manifest-file\n"
            "\n"
            "-b	- specify boot image to prepend\n"
-           "-r	- specify target root\n",
+           "-r	- specify target root\n"
+           "-s	- specify image file size; can be expressed in bytes, KB (with "
```

Comment:
May want to specify "minimum image file size", as the actual size could potentially be larger than the given size.

Reply:
OK, done